### PR TITLE
M2S-S2M Native ETH Configuration

### LINKED ISSUE #86: [M2S] Allow Configuration to Limit Native ETH Transfers
**Is your feature request related to a problem? Please describe.**
Yes, the setup of the SKALE Network gives every chain the same options with the Mainnet to SKALE Chain deposit boxes. 
However, the actual setup of liquidity for many of the mainnet SKALE Chains is utilizing the Europa Hub as an entry point for liquidity. 

From a user perspective, if they are attempting to use ETH on any other chain that is mapping to Europa; the user on Metaport has the option to bridge native ETH into any SKALE Chain.

**Describe the solution you'd like**
Similar to how Metaport only picks up ERC-20 tokens that are manually added for S2S; the default config for mainnet should require chain names in the config in order to transfer ETH in. Otherwise, it should not show up.

**Describe alternatives you've considered**
The only other alternative is updating IMA with a lock for this deposit box, however, the Metaport widget is still not doing any checks and assumes ETH can be bridged by default.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by TheGreatAxios**
- Added a custom configuration for ETH Native Tokens
	- Utilizes a chain list of strings
- On add token checks for inclusion in chain list

### CODE REVIEW

**Comment by TheGreatAxios** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by TheGreatAxios** [comment]
recheck

File: src/core/tokens/index.ts [hunk 1]
```diff
@@ -47,7 +47,8 @@ export async function getAvailableTokens(
     chainName1: string,
     chainName2: string,
     configTokens: interfaces.TokensMap,
-    autoLookup: boolean
+    autoLookup: boolean,
+    chains: string[]
```
Comment (payvint) [inline src/core/tokens/index.ts]: Looks like as an unnecessary change

Comment (TheGreatAxios) [reply src/core/tokens/index.ts]: Good catch. Will remove. Thanks 😀

**Review by payvint — COMMENTED** [review]
Nice looks good, let's wait for @dmitry-tk review!

**Review by payvint — APPROVED** [review]

**Review by dmytrotkk — APPROVED** [review]
